designer/client/src/brace/theme/nussknacker.css

```diff
@@ -10,6 +10,15 @@
     width: 1px;
     background: #555651
 }
+
+.ace-nussknacker .ace_search {
+    background: #818181;
+}
+
+.ace-nussknacker .ace_search_field {
+    color: #a8ce91;
```

Comment:
In the case of edit node fields, we use `#ccc` color, maybe it's worth it to be consistent with existing fields?

Reply:
Ok, changed.